docs/api/quickstart.md

```diff
@@ -0,0 +1,139 @@
+---
+title: API Quick Start Guide
+---
+
+The Rancher API is a programatic interface for your upstream and downstream clusters. Anything you can do through the Rancher GUI can also be performed through the commandline, using the Rancher API. This guide will help get you started on using the API from your client of choice.
```

Comment:
```suggestion
The Rancher API is a programatic interface for your upstream and downstream clusters. This guide will help get you started on using the API from your client of choice.
```

The second sentence won't be true for the initial API release since only subset of the CRDs will be available.

Reply:
Resolved in https://github.com/rancher/rancher-docs/pull/858/commits/aa74b3197d6945bdb7ff03039c6773c25a4d9020